modules/apps/data-engine/data-engine-rest-impl/src/main/java/com/liferay/data/engine/rest/internal/dto/v2_0/util/DataDefinitionUtil.java

```diff
@@ -26,6 +27,18 @@
  */
 public class DataDefinitionUtil {
 
+	public static String getContentType(DDMStructure structure) {
+		DataDefinitionContentType dataDefinitionContentType =
+			DataDefinitionContentTypeRegistryUtil.getDataDefinitionContentType(
+				structure.getClassNameId());
+
+		if (dataDefinitionContentType == null) {
+			return null;
+		}
+
+		return dataDefinitionContentType.getContentType();
+	}
```

Comment:
Why did you create another method? Are you using in another place?
If not, please revert as it was.

Reply:
Yes 

https://github.com/liferay-bpm/liferay-portal/blob/a20b3326e0c09650b202f3e3b4edd657c0352541/modules/apps/data-engine/data-engine-rest-impl/src/main/java/com/liferay/data/engine/rest/internal/dto/v2_0/util/DataDefinitionUtil.java#L60

https://github.com/liferay-bpm/liferay portal/blob/a20b3326e0c09650b202f3e3b4edd657c0352541/modules/apps/data-engine/data-engine-rest-impl/src/main/java/com/liferay/data/engine/rest/internal/resource/v2_0/DataDefinitionResourceImpl.java#L1360